src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -129,27 +130,15 @@ protected AbstractAuditLog(Settings settings, final ThreadPool threadPool, final
         restAuditingEnabled = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_REST, true);
         transportAuditingEnabled = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_TRANSPORT, true);
 
-        disabledRestCategories = new ArrayList<>(settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_DISABLED_REST_CATEGORIES, defaultDisabledCategories).stream()
-                .map(c->c.toUpperCase()).collect(Collectors.toList()));
-
-        if(disabledRestCategories.size() == 1 && "NONE".equals(disabledRestCategories.get(0))) {
-            disabledRestCategories.clear();
-        }
-
-        if (disabledRestCategories.size() > 0) {
-            log.info("Configured categories on rest layer to ignore: {}", disabledRestCategories);
-        }
+        disabledRestCategories = AuditCategory.parse(getConfigList(
```

Comment:
My understanding is that not all `AuditCategory` applies to Rest or Transport categories. It may be good to log a warning if an admin tries to disable auditing for a category that does not apply. Note that this case is different from an invalid category.

Reply:
Maybe raise another issue for this?